src/components/Navigation/NavigationItemTrigger/_navigationItemTrigger.scss

```diff
@@ -55,13 +55,20 @@
 
     &--hovered &__submenu {
       left: 0;
-      max-height: 700px;
+      max-height: calc(90vh - #{$header-height});
```

Comment:
Minor nitpick. Do we need the #{} around $header-height. The calc in the style above this doesn't need it so I'm just curious what this does for us.